test/unittest/unittest_nntrainer_internal.cpp

```diff
@@ -114,17 +114,18 @@ TEST(nntrainer_NeuralNetwork, load_config_03_n) {
 }
 
 /**
- * @brief Neural Network Model initialization
+ * @brief Neural Network Model initialization, there is warning when no
+ * input_shape is provided
  */
-TEST(nntrainer_NeuralNetwork, load_config_04_n) {
+TEST(nntrainer_NeuralNetwork, load_config_04_p) {
   int status = ML_ERROR_NONE;
   RESET_CONFIG("./test.ini");
   replaceString("Input_Shape = 32:1:1:62720", "", "./test.ini", config_str);
   nntrainer::NeuralNetwork NN;
   status = NN.setConfig("./test.ini");
   EXPECT_EQ(status, ML_ERROR_NONE);
   status = NN.loadFromConfig();
-  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
+  EXPECT_EQ(status, ML_ERROR_NONE);
```

Comment:
can you add a unittest here with `NN.init()` which fails when initializing a ini file based network with no input_shape?

Reply:
Actually, I'm making a parameterized, automated test for load config. I'll include that in that PR. Thanks

Reply:
ok.

Reply:
This should be dealt with #316